src/data-reporting-quicksight-stack.ts

```diff
@@ -187,6 +194,11 @@ export class DataReportingQuickSightStack extends Stack {
       value: dababoards,
     }).overrideLogicalId('Dashboards');
 
+    new CfnOutput(this, 'InternalUser', {
```

Comment:
manage the output name in constant to be shared with backend api

Reply:
this output is not necessary after move internal use creation out of quicksight stack.